interface/modules/custom_modules/oe-module-weno/templates/weno_fragment.php

```diff
@@ -0,0 +1,58 @@
+<?php 
```

Comment:
For all of your new scripts ensure have a standard header with author/license. Check out other scripts in OpenEMR for the standard template that is used.